packages/playback-core/src/index.ts

```diff
@@ -126,11 +157,19 @@ export const setupHls = (
       liveDurationInfinity: true,
     };
     const streamTypeConfig = getStreamTypeConfig(streamType);
+    const cmcd = experimentalCmcd
+      ? {
+          useHeaders: true,
+          sessionId: metadata.view_session_id,
+          contentId: metadata.video_id,
+        }
+      : undefined;
```

Comment:
NOTE: `hls.js` will get into a weird state if there is an empty object for `cmcd` in its options. We may want to open an issue for this on the hls.js repo, though it is a low priority.